### DISCUSSION TITLE
How to change to fontawesome for navigation icons?

### DISCUSSION THREAD

**Discussion by EriksonEnterprises** [filamentphp/filament#7766] [Help]
I tried the package called `owenvoke/blade-fontawesome` but I still get errors when trying to use fontawesome

I tried custom svgs, but since fontawesome has 3 categories (solid, regular, brands), I couldn't figure out how to use the sets properly as it kept trying to use default only for the navigation icons

When I did get it to work, the solid icons are completely black, so I can't see them in the navigation with dark mode, so maybe fontawesome isn't a good idea anyways

Also, I can't figure out how to change the color of icons either for the fontawesome icons. I got the solid black ones to work, so I thought maybe I can change the color to the panel color, but no way to do that where I could find..

Thank you

**Comment by bezhanSalleh** [comment]
once you installe the icon library of your choice you can follow the instructions here on how to replace them. lucky for you the example is for font-awesome icons.
https://filamentphp.com/docs/3.x/support/icons#replacing-the-default-icons
You can find the list of default icons used in filament here.
https://filamentphp.com/docs/3.x/support/icons#available-icon-aliases
